interface-spec.md

```diff
@@ -71,6 +71,16 @@ TODO: What if the literal has no language? Does it always have a datatype?
 
 TODO: Does the value always start with a question mark?
 
+### DefaultGraph extends Term
+
+An instance of `DefaultGraph` represents the default graph.
+It's only allowed to assign a `DefaultGraph` to the `.graph` property of a `Quad`. 
+
+**Properties:**
+
+- `String .termType` contains the constant `"defaultGraph"`.
+- `String .value` contains an empty string as constant value.
```

Comment:
Perhaps that `.equals` is true iff the other thing's `.termType` is `"defaultGraph"`?